### DISCUSSION TITLE
Ternary expressions

### DISCUSSION THREAD

**Discussion by andrewbanchich** [nushell/nushell#6432] [Q&A] [answered]
Are there any plans to support them or something similar?

What I'm trying to do is check if an env var exists, and if it does, return that. Otherwise return a literal "foo".

I can't seem to even figure out any relatively concise way to not error if the env var doesn't exist.

**Comment by andrewbanchich** [comment, marked as answer]
Actually, I think I answered my own question.

https://www.nushell.sh/book/commands/default.html#examples

**Reply by connormcmk** [reply]
How does that answer your question? 

Also, can't you do something ternary on a single line with something like `if x==true {x} else {y}`
